rnote-ui/src/workspacebrowser/mod.rs

```diff
@@ -43,6 +48,11 @@ mod imp {
         pub files_listview: TemplateChild<ListView>,
         pub files_dirlist: DirectoryList,
```

Comment:
I'm wondering if it's possible to split this up. Because currently the workspacebrowser gets bigger and bigger and it's in my opinion slowly getting a bit "too big" if you understand what I mean. Do you understand what I mean @flxzt